buildpacks/npm/lib/build.sh

```diff
@@ -34,6 +34,22 @@ fail_multiple_lockfiles() {
 	fi
 }
 
+store_node_version() {
+	local layers_dir=$1
+
+	if [[ -f "${layers_dir}/store.toml" ]]; then
+		local prev_node_version
+		# shellcheck disable=SC2002
+		prev_node_version=$(cat "${layers_dir}/store.toml" | grep node_version | xargs | cut -d " " -f3)
+
+		mkdir -p "${layers_dir}/env"
+		if [[ -s "${layers_dir}/env/PREV_NODE_VERSION" ]]; then
+			rm -rf "${layers_dir}/env/PREV_NODE_VERSION"
+		fi
+		echo -e "$prev_node_version" >>"${layers_dir}/env/PREV_NODE_VERSION"
+	fi
+}
```

Comment:
So I think what we want to do is either use the store.toml across each buildpack or use the PREV_NODE_VERSION from the nodejs-engine buildpack and share that through the build. Lets pair on this tomorrow though 😺